hathor/builder/cli_builder.py

```diff
@@ -103,6 +103,11 @@ def create_manager(self, reactor: Reactor) -> HathorManager:
             reactor_type=type(reactor).__name__,
         )
 
+        if not self._args.nano_testnet:
+            # Add protection to prevent enabling Nano Contracts due to misconfigurations.
+            self.check_or_raise(not settings.ENABLE_NANO_CONTRACTS,
+                                'configuration error: NanoContracts can only be enabled on localnets for now')
```

Comment:
If the user uses `--testnet` and `--nano-testnet`, this wouldn't raise an error, right?

Reply:
Fixed!